kotest-assertions/src/commonMain/kotlin/io/kotest/properties/gens.kt

```diff
@@ -638,12 +646,23 @@ fun <K, V> Gen.Companion.map(gen: Gen<Pair<K, V>>, maxSize: Int = 100): Gen<Map<
  * Returns the next pseudorandom, uniformly distributed value
  * from the ASCII range 33-126.
  */
-fun Random.nextPrintableChar(): Char {
-  val low = 32
-  val high = 127
-  return (nextInt(high - low) + low).toChar()
-}
+fun Random.nextPrintableChar(): Char = nextInt(from = 32, until = 127).toChar()
 
+/**
+ * Generates a [String] of [length] by calling [nextPrintableChar]
+ *
+ * ```kotlin
+ * // Examples
+ * val r = Random.Default
+ * r.nextPrintableString(-10) // ""
+ * r.nextPrintableString(0)   // ""
+ * r.nextPrintableString(1)   // " ", "a", "Z", etc.
+ * r.nextPrintableString(5)   // "Ha Ha", "gens!", "[{-}]", etc.
+ * ```
+ *
+ * @param length of String
+ * @return a given length String of random printable Chars
+ */
 fun Random.nextPrintableString(length: Int): String {
```

Comment:
Tried to highlight in the example that it's not just words that are getting generated here